ownCloudSDK/VFS/OCVFSNode.m

```diff
@@ -160,7 +161,11 @@ - (OCVFSItemID)vfsItemID
 
 	if ([vfsItemID isEqual:OCVFSItemIDRoot])
 	{
+		#if OC_FEATURE_AVAILABLE_FILEPROVIDER
 		return (NSFileProviderRootContainerItemIdentifier);
+		#else
+		return (OCVFSItemIDRoot);
+		#endif
```

Comment:
Is this code block change part of the translation or should it be moved to a extra PR?

Reply:
That should have went into `fix/reduce-mem-footprint` instead. (Re)moved it.